crates/chain-registry/src/querier.rs

```diff
@@ -107,6 +107,7 @@ impl QueryContext for SimpleHermesRpcQuerier {
             .map_err(|e| RegistryError::tendermint_url_parse_error(rpc_url.clone(), e))?;
 
         let client = HttpClient::builder(url)
+            .user_agent(format!("hermes/{}", ibc_relayer::HERMES_VERSION))
```

Comment:
Let's import the constant instead of using its fully qualified name

```suggestion
            .user_agent(format!("hermes/{}", HERMES_VERSION))
```

Reply:
Fixed